nautobot/dcim/tests/test_forms.py

```diff
@@ -159,11 +159,13 @@ def setUpTestData(cls):
 
     def test_interface_label_count_valid(self):
         """Test that a `label` can be generated for each generated `name` from `name_pattern` on InterfaceCreateForm"""
+        status_active = Status.objects.get_for_model(Interface).get(slug=InterfaceStatusChoices.STATUS_ACTIVE)
         interface_data = {
             "device": self.device.pk,
             "name_pattern": "eth[0-9]",
             "label_pattern": "Interface[0-9]",
             "type": InterfaceTypeChoices.TYPE_100ME_FIXED,
+            "status": status_active,
```

Comment:
Should this instead be?

```suggestion
            "status": status_active.pk,
```